fe/fe-core/src/main/java/com/starrocks/http/rest/TransactionResult.java

```diff
@@ -0,0 +1,28 @@
+// This file is made available under Elastic License 2.0.
```

Comment:
// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.